File: Javascript/EvaluateReversePolishNotation.js
var evalRPN = function(token){
    let stack = []

    for(const char of tokens){
        if(char === '+'){
            stack.push(stack.pop()+stack.pop())
        }
        else if(char === '-'){
            let num2 = stack.pop()
            let num1 = stack.pop()
            stack.push(num1-num2)
        }
        else if(char === '*'){
            stack.push(stack.pop()*stack.pop())
        }
        else if(char === '/'){
            let num2 = stack.pop()
            let num1 = stack.pop()
            stack.push(parseInt(num1/num2))
        }
        else{
            stack.push(parseInt(char))
        }
    }

    return stack[0]
}

File: Python/EvaluateReversePolishNotation.py
def evalRPN(tokens):
    
    stack = []

    for char in tokens:
        if char == '+':
            stack.append(stack.pop()+stack.pop())
        elif char == '-':
            num2 = stack.pop()
            num1 = stack.pop()
            stack.append(num1-num2)
        elif char == '*':
            stack.append(stack.pop()*stack.pop())
        elif char == '/':
            num2 = stack.pop()
            num1 = stack.pop()
            stack.append(int(num1/num2))
        else:
            stack.append(int(char))
    
    return stack[0]